### ISSUE TITLE
The start number of ordered list is not parsed correctly

### ISSUE THREAD

**Issue by lilingxi01** [aguingand/tiptap-markdown#51]
**Describe the bug**
When using the following content as initial value, the fourth item will not be starting at 10.

```
1. asldkfjal;sj
2. asld;kfjad;lsjf
3. asdfl;kjas;dlfjlak;j

10. a;sldkfjalksdf
```

The rendered content looks like this:

![CleanShot 2023-12-03 at 01 08 28@2x](https://github.com/aguingand/tiptap-markdown/assets/36816148/f6da36e3-173f-4d18-aea3-4d7bea645637)

**To Reproduce**
Use the above content as initial value.

**Expected behavior**
The expected index in [traditional markdown](https://www.markdownguide.org/basic-syntax/#ordered-lists) should actually be the 4 because traditional markdown does not separate ordered list no matter how many empty lines are in between.

But the Tiptap supports two adjacent ordered lists to have unconnected numbers because they are treated as two separate nodes rather than just one node. I think it would be good to align with the support of Tiptap or at least provide an option to override that.

![CleanShot 2023-12-03 at 01 19 08@2x](https://github.com/aguingand/tiptap-markdown/assets/36816148/0b020044-47c8-4c74-96f5-c8f798934759)

**Desktop (please complete the following information):**
Not related

**Smartphone (please complete the following information):**
Not related

**Additional context**
I can help with the PR if you think it is good to support.

**Comment by aguingand** [comment]
Released in 0.8.8. Adjacent ordered list are now rendered like this:
```
1. foo

1) bar

1. baz
```

**Comment by lilingxi01** [comment]
Thank you! This is fast!!
